db/R/dbfiles.R

```diff
@@ -147,18 +147,17 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
 ##' @param con database connection object
 ##' @param hostname the name of the host where the file is stored, this will default to the name of the current machine
 ##' @param params database connection information
-##' @param exact.dates setting to include start and end date in input query
 ##' @return data.frame with the id, filename and pathname of the input that is requested
 ##' @export
-##' @author Rob Kooper, Tony Gardella
+##' @author Rob Kooper
 ##' @examples
 ##' \dontrun{
 ##'   dbfile.input.check(siteid, startdate, enddate, 'application/x-RData', 'traits', dbcon)
 ##' }
 dbfile.input.check <- function(siteid, startdate=NULL, enddate=NULL, mimetype, formatname, parentid=NA, 
-                               con, hostname=fqdn(), exact.dates=FALSE) {
+                               con, hostname=fqdn(), ignore.dates=FALSE) {
   if (hostname == "localhost") hostname <- fqdn();
```

Comment:
This is a mistake and I must be out of sync. I still don't understand how though because my pull from pecan master doesn't update anything.